### PR TITLE
feat: uopool unsafe

### PR DESCRIPTION
**Pull request by Vid201**

### CODE REVIEW

File: crates/primitives/src/uopool.rs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#[derive(clap::ValueEnum, Clone, Copy, Debug, PartialEq, Eq)]
+pub enum Mode {
+    Auto,
```
Comment (zsluedem) [inline crates/primitives/src/uopool.rs:3]: `Auto` seems ambiguous to me. Any better words?

**Review by zsluedem — APPROVED** [review]
Generally good

File: crates/primitives/src/uopool.rs (hunk 1 above)
Comment (Vid201) [reply crates/primitives/src/uopool.rs:3]: I was thinking about "Standard". Do you have any other suggestion?

**Comment by Vid201** [comment]
> Generally good

We will need to refactor verification of user operations for alt mempools, since we will have then normal, unsafe, altmempool1, altmempool2 ...

**Pull request merged**